Storage/SQL/SQLiteHistoryRecommendations.swift

```diff
@@ -171,36 +174,35 @@ extension SQLiteHistory: HistoryRecommendations {
         return [(clearHighlightsQuery, nil), (sql, args)]
     }
 
-    // Checks if there are more than 100k items in the `visits` table.
+    // Checks if there are more than 250k items in the `history` table.
     // This is used as an indicator that we should clean up old history.
     func checkIfCleanupIsNeeded() -> Deferred<Maybe<Bool>> {
-        let sql = "SELECT COUNT(*) FROM \(TableVisits)"
+        let sql = "SELECT COUNT(*) FROM \(TableHistory)"
```

Comment:
You should check whether `SELECT COUNT(rowid)` has a better query plan than `SELECT COUNT(*)`. There's conflicting information in the docs.

Reply:
No difference in the query plan between `rowid` and `*`. I changed it to `rowid` anyway.